### PR TITLE
Add Missing typeTags

### PR DESCRIPTION
**Pull request by koosemose**
Adds typeTags to Airlock and Metal Smelter.

### CODE REVIEW

File: Assets/StreamingAssets/Data/Furniture.xml [hunk 1]
```diff
@@ -62,6 +62,7 @@
 
     <Furniture objectType="Airlock">
         <Name>Door</Name>
+		<TypeTag>Door</TypeTag>
```
Comment (Mizipzor) [inline Assets/StreamingAssets/Data/Furniture.xml]: These are tabs. :)

**Comment by RodRitter** [comment]
If you fix the styling issue, I'll merge this.

**Comment by koosemose** [comment]
Gah, no idea how those got in there, they're strangely not present in other places where I've edited. Anyways, fixing those now. Would it be reasonable to also fix other tabs that seem to have snuck through in furniture.xml?

**Comment by RodRitter** [comment]
Yea that should be fine, if you don't mind cleaning it up.

**Comment by Mizipzor** [comment]
Yes, absolutely. Everyone seems to agree that you cant fix stuff like that everywhere in one go. Too difficult to merge. So while youre in a file anyway, if you have the time clean some of the obvious stuff up. Thats what Ive been doing. :)

**Comment by koosemose** [comment]
Goodness there were 64 tabs in there.

**Comment by koosemose** [comment]
All fixed up now.

**Comment by RodRitter** [comment]
Great :)

**Comment by Mizipzor** [comment]
Did you count? :P

By the way, I made [a script](https://gist.github.com/Mizipzor/4da24230e31206650bff2f970a7093f3) for doing this automatically.

**Comment by koosemose** [comment]
Nope, I used find and replace in Notepad++, it counted for me. And I don't think I'm set up properly with Python anyways, anything I would want to do with it I can do with perl before I figure out the mess of my Python install.

**Pull request merged**